test/spec/Feature/Auth/JwtCacheSpec.hs

```diff
@@ -0,0 +1,22 @@
+module Feature.Auth.JwtCacheSpec where
+
+import Network.Wai (Application)
+
+import Test.Hspec
+import Test.Hspec.Wai
+import Test.Hspec.Wai.JSON
+
+import Protolude hiding (get)
+
+spec :: SpecWith ((), Application)
+spec = describe "jwt cache enabled" $ do
+  it "denies access to tables that anonymous does not own" $
+    get "/authors_only" `shouldRespondWith`
+      [json| {
+        "hint":null,
+        "details":null,
+        "code":"42501",
+        "message":"permission denied for table authors_only"} |]
+      { matchStatus = 401
+      , matchHeaders = ["WWW-Authenticate" <:> "Bearer"]
+      }
```

Comment:
PR looks great overall :+1: 

However this test puzzles me, why is it needed here? Isn't this the same as:

https://github.com/PostgREST/postgrest/blob/4348cb205738a8497fadcaed4ab6015aa465b87d/test/spec/Feature/Auth/AuthSpec.hs#L17-L26

Reply:
Yes, it's the same but written for code-coverage with `jwt cache: enabled` and `server-timing: enabled|disabled`. I should better mention this in the comments.

Reply:
And the rest of the tests related to the JWT cache are io tests in python right? Maybe mention that in the comments. Also, for consistency why not make this an io test? Otherwise you have two files to look at for jwt cache tests.

Reply:
> Also, for consistency why not make this an io test? Otherwise you have two files to look at for jwt cache tests.

Agreed. I was thinking the same.